lmdeploy/cli/serve.py

```diff
@@ -226,6 +249,21 @@ def add_parser_proxy():
                             help='Whether to disable cache status of the '
                             'proxy. If set, the proxy will forget the status '
                             'of the previous time')
+        # For Disaggregation
+        parser.add_argument('--migration-protocol',
+                            type=str,
+                            choices=['TCP', 'RDMA', 'NVLINK'],
+                            default='RDMA',
+                            help='transport protocol of KV migration')
+        parser.add_argument('--link-type',
+                            type=str,
+                            choices=['Ethernet', 'IB'],
+                            default='Ethernet',
+                            help='RDMA Link Type')
+        parser.add_argument('--disable-gdr',
```

Comment:
is `gdr` short for `GPU Direct Memory Access`?
If open the option`--disable-gdr`, what `--link-type` should we choose?